src/Sylius/Behat/Context/Ui/EmailContext.php

```diff
@@ -48,22 +48,25 @@ public function __construct(SharedStorageInterface $sharedStorage, EmailCheckerI
      */
     public function anEmailShouldBeSentTo($recipient)
     {
-        $this->assertEmailHasRecipient($recipient);
+        Assert::true(
+            $this->emailChecker->hasRecipient($recipient),
+            sprintf('An email should have been sent to %s.', $recipient)
+        );
     }
 
     /**
      * @Then :count email(s) should be sent to :recipient
      */
     public function numberOfEmailsShouldBeSentTo($count, $recipient)
     {
-        $this->assertEmailHasRecipient($recipient);
+        $actualMessagesCount = $this->emailChecker->countMessagesTo($recipient);
 
         Assert::eq(
```

Comment:
Won't work as the `$count` we get is a string, while `$actualMessagesCount` is an int.
Ofc `$count` could be transformed to an int, but personally i think that'd be a tad of an overkill.